create/application.go

```diff
@@ -162,9 +172,25 @@ func (app *applicationCmd) Run(ctx context.Context, client *api.Client) error {
 }
 
 func (app *applicationCmd) config() apps.Config {
+	var deployJob *apps.DeployJob
+
+	if len(app.DeployJob.Command) != 0 {
+		deployJob = &apps.DeployJob{
+			Job: apps.Job{
+				Name:    app.DeployJob.Name,
```

Comment:
Yeah the name always has to be set, else the deploy job will break. I could not find a way to validate this via kong, so `name` now also has to be set to something non-empty or it will not create a deploy job.